content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -32,9 +32,10 @@ GPU vendor:
 
 * [AMD](#deploying-amd-gpu-device-plugin)
 * [NVIDIA](#deploying-nvidia-gpu-device-plugin)
+* [Intel](#deploying-intel-gpu-device-plugin)
 
-When the above conditions are true, Kubernetes will expose `amd.com/gpu` or
-`nvidia.com/gpu` as a schedulable resource.
+When the above conditions are true, Kubernetes will expose `amd.com/gpu`,
+`nvidia.com/gpu` or `gpu.intel.com/i915` as a schedulable resource.
```

Comment:
Is it _always_ `i915`?

Reply:
Indeed the Intel GPU resource is for legacy reasons currently named as `i915`.

There is an exception to that in case the user configures our plugin with "monitoring" enabled. Then an additional single `i915_monitoring` resource is also created, allowing access to all GPUs. That is intended to be used by prometheus metrics collectors (collectd, or intel xpumanager).

https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/cmd/gpu_plugin/README.md#modes-and-configuration-options

Reply:
Another option: have a page in Intel's own docs about the bug, and link there instead.

Reply:
> Another option: have a page in Intel's own docs about the bug, and link there instead.

Like this one, which describes the resource names for all our Kubernetes device plugins? https://github.com/intel/intel-device-plugins-for-kubernetes#workload-authors

I could add a note to our section in this page with an explanation that the Intel plugin resource naming is different, with a link there.

Granted, we have not considered our naming scheme as a bug, rather more like a feature with legacy. Now if our resource naming is problematic for the Kubernetes documentation, we could leave out the change at lines 37 and 38, I'm optimistic the users can figure it out with the documentation and examples which we have.

The intent with this PR here is to help Kubernetes users find our GPU plugin sources, similar to the GPU plugins of AMD and NVIDIA, which also have their respective links to their implementations. The details then, are potentially too many to duplicate in this Kubernetes page.

Reply:
We could link to https://github.com/intel/intel-device-plugins-for-kubernetes, sure. It's even better if there is an actual documentation site.

Bug https://github.com/kubernetes/website/issues/36983 is relevant too. The ideal situation is to cut as much vendor-specific detail out of the page, and replace it with links to vendors' own docs.

Reply:
> We could link to https://github.com/intel/intel-device-plugins-for-kubernetes, sure. It's even better if there is an actual documentation site.
> 
> Bug #36983 is relevant too. The ideal situation is to cut as much vendor-specific detail out of the page, and replace it with links to vendors' own docs.

We do have an actual documentation site also. The content is exactly the same since it is generated from the sources, but browsing through it is a bit nicer thanks to the layout and side panel TOC. If that would be your preference, we could point the users to it. It is here: https://intel.github.io/intel-device-plugins-for-kubernetes/README.html and the GPU part direct link is here: https://intel.github.io/intel-device-plugins-for-kubernetes/cmd/gpu_plugin/README.html

The reason why I mentioned in the PR those GPU sharing functionalities of our implementation is that unlike what is said currently in this documentation page, GPU sharing is possible with our implementation. And if I'm not mistaken, it nowadays is possible also with the NVIDIA GPU plugin, based on e.g. https://github.com/kubernetes/kubernetes/issues/52757#issuecomment-1245796945 which then would open the door to actually remove the statements about GPU sharing entirely from this page and leave it as a vendor specific detail. What would you think?

Reply:
> Bug https://github.com/kubernetes/website/issues/36983 is relevant too.

Lack of 3rd party warning is a pre-existing issue.  You need a separate PR for it, not this one.

(As to vendor specific details, IMHO you should file separate bug for that instead of cramming extra things as comments to previously well-focused bug.)

Reply:
> actually remove the statements about GPU sharing entirely from this page and leave it as a vendor specific detail

Maybe do that as a first step, and then see what's left to do for this issue.

Reply:
> > actually remove the statements about GPU sharing entirely from this page and leave it as a vendor specific detail
> 
> Maybe do that as a first step, and then see what's left to do for this issue.

The later commits which I pushed into this PR based on our discussion here do just that. Do take a look. I could, if need be, split that GPU sharing text removal patch into a separate PR of course.

I can only provide the Intel section in a more or less clean fashion which just points to our documentation. Perhaps pointing to just the main page of our device plugins documentation site would be the most robust choice, because links to the documentation site internals are more likely to break, if documentation gets updated.

As to the other vendor's details, I feel that I'm in a bad position to make changes in those sections.